src/chrome/content/rules/MySQL.xml

```diff
@@ -59,6 +60,25 @@
 							-->
 	<securecookie host="^\.mysql\.\w{2,3}$" name="^MySQL_S$"/>
 
+
+	<test url="http://cdn.mysql.com/Downloads/Connector-J/mysql-connector-java-5.1.34.tar.gz.md5" />
+	<test url="http://cdn.mysql.com/Downloads/MySQL-5.6/MySQL-5.6.23-1.linux_glibc2.5.i386.rpm-bundle.tar.md5" />
+
+	<!--  http://downloads.mysql.com/ -> https://dev.mysql.com/downloads/
+	      Other pages like http://downloads.mysql.com/archives/
+              don't have https pages
+	-->
+	<exclusion pattern="^http://downloads\.mysql\.com/." />
```

Comment:
This will fail the recently-added 'unescaped dot in exclusion' test in trivial-validate.py. Change it to \w?